### DISCUSSION TITLE
Import any file without transforming

### DISCUSSION THREAD

**Discussion by apfelbox** [parcel-bundler/parcel#7780] [Q&A] [answered]
Hi,

I just want to import a file without any transformation. I just want to get the file path to use it as image / video.
Is that possible? Or do you always need a transformer to import a file?

**Comment by mischnic** [comment, marked as answer]
Are you looking for something like this? https://parceljs.org/languages/javascript/#url-dependencies

**Reply by apfelbox** [reply]
Apparently... yes! Just using `import file from 'url:...';` indeed just works. Thanks 🙌
